src/openloco/localisation/languagefiles.cpp

```diff
@@ -200,6 +200,9 @@ namespace openloco::localisation
                 }
                 else if (commands[0] == "MOVE_X")
                 {
+                    *out++ = (char)control_codes::move_x;
+                    uint8_t pixels_to_move_by = std::atoi(commands[1].data());
```

Comment:
> Unlike std::basic_string::data() and string literals, data() may return a pointer to a buffer that is not null-terminated.  (https://en.cppreference.com/w/cpp/string/basic_string_view/data)

Reply:
Fairly sure that doesn't matter for `atoi`, though, as it's only parsing [0-9].
> Discards any whitespace characters until the first non-whitespace character is found, then takes as many characters as possible to form a valid integer number representation and converts them to an integer value. The valid integer value consists of the following parts: 
> * (optional) plus or minus sign
> * numeric digits